Modifications for 32-bit Linux installations.

Default is for 64 bit installations.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -126,7 +126,7 @@
 #Banner none
 
 # override default of no subsystems
-Subsystem	sftp	/usr/lib64/misc/sftp-server
+Subsystem	sftp	/usr/lib/misc/sftp-server
 
 # the following are HPN related configuration options
 # tcp receive buffer polling. disable in non autotuning kernels
